qiskit_nature/results/protein_folding_result.py

```diff
@@ -60,3 +98,29 @@ def get_result_binary_vector(self) -> str:
             result.append(self._best_sequence[index])
 
         return "".join(result[::-1])
+
+    def get_xyz_file(self, name: str = None, output_data: bool = False) -> np.ndarray:
```

Comment:
Should this not be called `get_xyz_data` since by default it does not create an xyz file. And why not have a separate `save_xyz_file` that calls the former and saves it to the given name. Yes one would need to call both to do both things but I find the "side-effect" of creating a file above when getting the data "unsettling"